lib/Core/Site/Values/Content.php

```diff
@@ -36,6 +54,11 @@ public function __construct(array $properties = [])
             unset($properties['_fields_data']);
         }
 
+        if (array_key_exists('site', $properties)) {
+            $this->site = $properties['site'];
```

Comment:
It has to be this way, this extends from eZ's [`ValueObject`](https://github.com/ezsystems/ezpublish-kernel/blob/master/eZ/Publish/API/Repository/Values/ValueObject.php), and `$site` property here is private. So I can't pass it to the parent constructor because it would fail in the magic setter for not being visible there, and if I were to make it `protected`, it would be readable, which is a no-go.